apartments/tests.py

```diff
@@ -37,7 +37,7 @@ def new_apartment(new_user, new_city):
         start_date='2021-1-1',
         about='Hey!',
         image_url='www.some-url.com',
-    )
+        )
     apartment.save()
```

Comment:
I think that create is saving already

Reply:
Good to know.. I will change that.
Thanks